packages/components/src/Gallery/Gallery.css

```diff
@@ -0,0 +1,58 @@
+.gallery {
+  display: flex;
+  flex-wrap: wrap;
+  gap: var(--space-base);
+}
+
+.card {
+  --gallery-card--size: var(--space-largest);
+  width: var(--gallery-card--size);
+  height: var(--gallery-card--size);
+}
+
+.image {
+  width: 100%;
+  height: 100%;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+/* Overriding border styling to match card  */
+.border {
+  --color-border: var(--color-grey--lighter);
```

Comment:
Our semantic [border color](https://atlantis.getjobber.com/colors#border) already maps to `grey--lighter`